raiden/tests/utils/smoketest.py

```diff
@@ -295,7 +289,7 @@ def complete_genesis():
 
 
 def init_with_genesis(smoketest_genesis):
-    with open(GENESIS_PATH, 'wb') as handler:
+    with open(GENESIS_PATH, 'w') as handler:
```

Comment:
why was the binary option removed? (just curious, `\n` vs `\r\n` should be a problem for json anyways)

Reply:
this is a json file, so it should be text, not binary